x-pack/test/api_integration/apis/fleet/agents/enroll.ts

```diff
@@ -23,7 +23,7 @@ export default function (providerContext: FtrProviderContext) {
     before(async () => {
       await esArchiver.loadIfNeeded('fleet/agents');
 
-      const { body: apiKeyBody } = await esClient.security.createApiKey({
+      const { body: apiKeyBody } = await esClient.security.createApiKey<typeof apiKey>({
```

Comment:
TIL:  the `elastic/elasticsearch-js` package was already used in dev. As I bumped the version, I had to fix a few usages.